src/providers/twitch/TwitchChannel.cpp

```diff
@@ -927,6 +935,76 @@ void TwitchChannel::refreshCheerEmotes()
         .execute();
 }
 
+void TwitchChannel::createClip()
+{
+    if (!this->isLive())
+    {
+        this->addMessage(makeSystemMessage(
+            "Cannot create clip while the channel is offline!"));
+        return;
+    }
+
+    if (QTime().currentTime() < this->timeNextClipCreationAllowed_ ||
+        this->isClipCreationInProgress)
+    {
+        return;
+    }
```

Comment:
Should we also post a system message here informing the user that a clip creation is currently in progress or that they must wait x more seconds before being able to create a new clip?

Reply:
I didn't want to have unnecessary message while clip creation is in progress since it's rather straight-forward that you can't create other clips in this short period of time.
I'm unsure about cases when you are on "clip cooldown". I thought about maybe adding some information in FAQ about it or in changelog entry.

Reply:
Yeah, I'm not sure about this one either 👍 I can agree with both ways of going about it